homework-projects/rupesh_final/template-index.adoc

```diff
@@ -0,0 +1,10 @@
+[[index-file-template]]
+= Peek Index
+:toc: left
+:toclevels: 3
+
+This index file template contains two included files on Peek: Screen Recording Tool
+ 
+include::template-included-file-1.adoc[leveloffset=+1]

```

Comment:
Rename the file to something else than template.

Reply:
done